### PR TITLE
"Remove User" button and popup added - G3 2024 v6 #15425

### PR DESCRIPTION
**Pull request by a22natth**
In the example below, an additional icon for removing a user has been added. 
Pasted Graphic 12

When clicking the icon, the popup below will be presented.
Pasted Graphic 13

The field to the right is a dropdown which also acts as a search bar for different users. 
Pasted Graphic 14

In collaboration with @a18chrkr

### CODE REVIEW

File: DuggaSys/accessed.php [hunk 1]
```diff
@@ -123,16 +123,80 @@
...
+							<option value="User 1"></option>
+							<option value="User 2"></option>
+							<option value="User 3"></option>
+						</datalist>
+					</div>
+					<div class="flex-end">
+						<input class='submit-button' type='button' value='Save' onclick='addSingleUser();' />
+					</div>
+				</div>
+      		</div>
```
Comment (a22willi) [inline DuggaSys/accessed.php:149]: Missed closing div or indentation error.

File: DuggaSys/accessed.php [hunk 2]
```diff
@@ -123,16 +123,80 @@
...
```
Comment (a22willi) [inline DuggaSys/accessed.php:176]: Missed closing div or indentation error.

File: DuggaSys/accessed.php [hunk 3]
```diff
@@ -123,16 +123,80 @@
...
+     	</div>
+	</div>
```
Comment (a22willi) [inline DuggaSys/accessed.php:178]: Indentation

File: DuggaSys/accessed.php [hunk 4]
```diff
@@ -123,16 +123,80 @@
...
+			</li>
 	        <li onclick="showCreateUserPopup();">
-							<a id="cFabBtn" class="btn-floating fab-btn-sm scale-transition scale-out" data-tooltip='Create user'>
-									<img id="cFabBtnImg" class="fab-icon" src="../Shared/icons/createUser.svg">
-							</a>
-					</li>
+				<a id="cFabBtn" class="btn-floating fab-btn-sm scale-transition scale-out" data-tooltip='Create user'>
+					<img class="fab-icon" src="../Shared/icons/createUser.svg">
+				</a>
+			</li>
```
Comment (a22willi) [inline DuggaSys/accessed.php:199]: Indentation.

File: Shared/css/style.css [hunk 5]
```diff
@@ -9698,12 +9698,11 @@ margin:2px;
...
 }
 
-#createUser .flexwrapper {
-  /* margin-top: .75rem; */
+#createUser .flexwrapper, #addUser .flexwrapper, #removeUser .flexwrapper  {
   margin-bottom: .75rem;
 }
 
-#createUser .flex-end{
+#createUser .flex-end, #addUser .flex-end, #removeUser .flex-end{
```
Comment (a22willi) [inline Shared/css/style.css:9705]: .flex-end added twice.

**Review by a22willi — CHANGES_REQUESTED** [review]
The dropdown content does not show up for me unless I click the textbox/dropdown box twice.

**Review by a22willi — APPROVED** [review]
After talking with @a22natth and @a18chrkr it seems that the indentation looks fine in visual studio. And that the search box behavior is working as intended, with slight browser based variations.

File: DuggaSys/accessed.php [hunk 6]
```diff
@@ -123,16 +123,80 @@
...
+			<div>
+				<div class='loginBoxheader'>
+					<h3>Add user</h3>
+					<div class='cursorPointer' onclick='closeWindows();'>x</div>
+				</div>
+				<div class="content-wrapper">
+					<input type='hidden' id='uid' value='Toddler' />
+					<div class='flexwrapper'>
+						<span>Choose user:</span>
+						<div class="tooltipDugga"><span id="tooltipFirst" style="display: none;" class="tooltipDuggatext">  </span></div>
```
Comment (a22moamy) [inline DuggaSys/accessed.php:137]: Unnecessary inline styling :(

File: DuggaSys/accessed.php [hunk 7]
```diff
@@ -123,16 +123,80 @@
...
+					<div class="flex-end">
+						<input class='submit-button' type='button' value='Save' onclick='addSingleUser();' />
+					</div>
+				</div>
+      		</div>
+     	</div>
+	</div>
+
+	<div id='removeUser' class='loginBoxContainer' style='display:none;'>
+		<div class='loginBox' style='width:494px;'>
```
Comment (a22moamy) [inline DuggaSys/accessed.php:154]: Unnecessary inline styling :(

File: DuggaSys/accessed.php [hunk 8]
```diff
@@ -123,16 +123,80 @@
...
+			<div>
+				<div class='loginBoxheader'>
+					<h3>Remove user</h3>
+					<div class='cursorPointer' onclick='closeWindows();'>x</div>
+				</div>
+				<div class="content-wrapper">
+					<input type='hidden' id='uid' value='Toddler' />
+					<div class='flexwrapper'>
+						<span>Choose user:</span>
+						<div class="tooltipDugga"><span id="tooltipFirst" style="display: none;" class="tooltipDuggatext">  </span></div>
```
Comment (a22moamy) [inline DuggaSys/accessed.php:164]: Unnecessary inline styling :(

File: DuggaSys/accessed.php [hunk 9]
```diff
@@ -123,16 +123,80 @@
 		</div>
 	</div>
 
+	<div id='addUser' class='loginBoxContainer' style='display:none;'>
+		<div class='loginBox' style='width:494px;'>
```
Comment (a22moamy) [inline DuggaSys/accessed.php:127]: Unnecessary inline styling :(

File: DuggaSys/accessed.php [hunk 10]
```diff
@@ -123,16 +123,80 @@
 		</div>
 	</div>
 
+	<div id='addUser' class='loginBoxContainer' style='display:none;'>
```
Comment (a22moamy) [inline DuggaSys/accessed.php:126]: Unnecessary inline styling :(

File: Shared/css/style.css (hunk 5 above)
Comment (a22moamy) [reply Shared/css/style.css:9708]: this has still not been solved

**Review by a22moamy — APPROVED** [review]
code looks mostly good now :)

**Review by a22willi — APPROVED** [review]
Still works after removing inline css.

**Pull request merged**